### PR TITLE
2489 finish account button

### PR DESCRIPTION
**Pull request by Baron-Severin**
Connected to #???

## Checklist

- [X] Confirm the  **acceptance criteria is/are fully satisfied** in the issue(s) this PR will close
- [ ] Add **testing notes and/or screenshots** in PR description to help guide all potential reviewers
- [ ] Add thorough **tests** or an explanation of why it does not
- [X] Add a **CHANGELOG entry** if applicable
Not user facing
- [ ] Add **QA labels** on the associated issue (not this PR; `qa-ready` or `qa-denied`)

### CODE REVIEW

**Comment by Baron-Severin** [comment]
@liuche 

# Request for data collection review form

1) What questions will you answer with this data?

- How many users begin the FxA login process?

2) Why does Mozilla need to answer these questions?  Are there benefits for users? Do we need this information to address product or business requirements?

* To see how many users begin logging in vs how many complete the process

3) What alternative methods did you consider to answer these questions? Why were they not sufficient?

- This is the only way to determine this information.

4) Can current instrumentation answer these questions?

- No telemetry currently exists to answer these questions.

5) List all proposed measurements and indicate the category of data collection for each measurement, using the Firefox [data collection categories](https://wiki.mozilla.org/Firefox/Data_Collection) on the Mozilla wiki.   

| Measurement Description                      | Data Collection Category | Tracking Bug #                                               |
| -------------------------------------------- | ------------------------ | ------------------------------------------------------------ |
| User has clicked on the FxA login button | 2                        | [Issue 2489](https://github.com/mozilla-mobile/firefox-tv/issues/2489) |

6) How long will this data be collected?

* @athomasmoz will permanently monitor this data.

7) What populations will you measure?

* All users with telemetry enabled, on release channel.


8) If this data collection is default on, what is the opt-out mechanism for users?

- There is an option in settings to stop sending usage data.

9) Please provide a general description of how you will analyze this data.

- The data will be analyzed through Redash queries and dashboards.

10) Where do you intend to share the results of your analysis?

- On Redash and with mobile teams.

File: app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt [hunk 1]
```diff
@@ -117,7 +118,10 @@ class NavigationOverlayFragment : Fragment() {
             }
 
             // TODO: change button action based on profile state.
-            NavigationEvent.FXA_BUTTON -> serviceLocator.fxaLoginUseCase.beginLogin(fragmentManager!!)
+            NavigationEvent.FXA_BUTTON -> {
```
Comment (liuche) [inline app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt]: Can you also add this to the `docs/telemetry.md` file?

File: app/src/main/java/org/mozilla/tv/firefox/telemetry/TelemetryIntegration.kt [hunk 2]
```diff
@@ -88,6 +88,7 @@ open class TelemetryIntegration protected constructor(
         const val DESKTOP_MODE = "desktop_mode"
         const val VIDEO_ID = "video_id"
         const val FULL_SCREEN_VIDEO = "full_screen_video"
+        const val FXA = "fxa"
```
[the remark this answers is not in the document]
Comment (Baron-Severin) [reply app/src/main/java/org/mozilla/tv/firefox/telemetry/TelemetryIntegration.kt]: Talked about this IRL

File: app/src/main/java/org/mozilla/tv/firefox/telemetry/TelemetryIntegration.kt [hunk 3]
```diff
@@ -107,6 +108,7 @@ open class TelemetryIntegration protected constructor(
         const val SETTINGS_SEND_DATA_TILE = "send_data_tile"
         const val SETTINGS_ABOUT_TILE = "about_tile"
         const val SETTINGS_PRIVACY_TILE = "privacy_tile"
+        const val FXA_LOGIN_BUTTON = "fxa_login_button"
```
[the remark this answers is not in the document]
Comment (Baron-Severin) [reply app/src/main/java/org/mozilla/tv/firefox/telemetry/TelemetryIntegration.kt]: Talked about this IRL

File: app/src/main/java/org/mozilla/tv/firefox/telemetry/TelemetryIntegration.kt [hunk 4]
```diff
@@ -287,6 +289,11 @@ open class TelemetryIntegration protected constructor(
         TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.SETTING, telemetryValue).queue()
     }
 
+    // TODO send different values depending on the Fxa state
```
[the remark this answers is not in the document]
Comment (Baron-Severin) [reply app/src/main/java/org/mozilla/tv/firefox/telemetry/TelemetryIntegration.kt]: It seems to be, there's a comment from #2445 that says `// TODO: change button action based on profile state.`.  I've added an addition comment beneath it that reminds to update telemetry as well.

**Pull request merged**